projects/storefrontstyles/scss/components/content/tab-paragraph-container/_tab-paragraph-container.scss

```diff
@@ -1,4 +1,51 @@
 $useTabsOnPDP: false !default;
+$useAccordionOnly: false !default;
+
+@mixin accordion {
+  > button {
+    border-top: 1px var(--cx-color-light) solid;
+    font-size: 18px;
```

Comment:
font-size: 18px should be change to a variable. `font-size: var(--cx-font-size, 1.125rem);`